packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -51,6 +51,7 @@ If you need the proxy smart contract addresses, we list the most relevant ones b
 "rinkeby": {
 	"address": "0x162edb802fae75b9ee4288345735008ba51a4ec9",
 }
+// FIXME
```

Comment:
?

Reply:
It was regarding goerli, I added `FIXME: add goerli`